libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -586,6 +586,7 @@ struct PACKED Packed_Location_Option_Flags {
     uint8_t terrain_alt  : 1;           // this altitude is above terrain
     uint8_t origin_alt   : 1;           // this altitude is above ekf origin
     uint8_t loiter_xtrack : 1;          // 0 to crosstrack from center of waypoint, 1 to crosstrack from tangent exit location
+    uint8_t local_frame : 1;            // 1 if lat/lng is N/E postion vector in centimeters relative to ekf origin
```

Comment:
this changes the binary format of our mission storage. I am concerned that this could break existing missions in storage on stm32 or linux boards to re-interpret